client/src/components/Creator.js

```diff
@@ -10,17 +19,42 @@ class Creator extends Component {
     this.state = {
       ETHinUSD: "0",
       aUSDCBalance: "0",
+      sentTransactionHash: "",
+      aaveRate: "0",
+      estimatedFuture: "0",
+      copied: false,
+      ethTransactions: [],
     };
   }
 
   getLink(hash) {
     return "https://kovan.etherscan.io/tx/" + hash;
   }
 
+  cashOut() {
+    if (this.state.aUSDCBalance === "0") {
+      alert("You don't have any funds to cash out");
+    } else {
+      alert(
+        "You've successfully withdrawn your aUSDC and will show up as USDC in your wallet."
+      );
+      this.setState({ aUSDCBalance: "0", estimatedFuture: "0" });
+    }
+  }
+
+  calculateInterest(time) {
+    const principle = parseFloat(this.state.aUSDCBalance);
+    const rate = parseFloat(this.state.aaveRate);
+    const timeYears = parseInt(time) / 12;
+
+    const A = principle * (1 + rate * timeYears);
```

Comment:
nit: Can we just `return principle * (1 + rate * timeYears);`? (not sure what A stands for too)